src/NuGetForUnity/Editor/PathHelper.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Text;
+using UnityEngine;
+
+/// <summary>
+///     Helper class to handle path operations.
+/// </summary>
+internal static class PathHelper
+{
+    /// <summary>
+    ///     The path comparison type to use for the current platform the editor is running on.
+    /// </summary>
+    internal static readonly StringComparison PathComparisonType =
+        Application.platform == RuntimePlatform.WindowsEditor || Application.platform == RuntimePlatform.OSXEditor ?
+            StringComparison.OrdinalIgnoreCase :
+            StringComparison.Ordinal;
+
+    /// <summary>
+    ///     Create a relative path from one path to another. Paths will be resolved before calculating the difference.
+    ///     Default path comparison for the active platform will be used (OrdinalIgnoreCase for Windows or Mac, Ordinal for Unix).
+    /// </summary>
+    /// <remarks>
+    ///     Should the project upgrade to Unity version that has .NET Standard 2.1, this can be removed and instead we will use:
+    ///     Path.GetRelativePath(relativeTo, path).
```

Comment:
We actually can't use Path.GetRelativePath even then since it will not work correctly when calling the CLI version with a project path passed as argument that is different from current directory.

Reply:
see other comment